src/synth.ts

```diff
@@ -1,10 +1,28 @@
 // Classic variants are from Scale Workshop 2, they're softer on SW3.
 
 import { AperiodicWave } from 'sw-synth'
-import { centsToValue, valueToCents } from 'xen-dev-utils'
+import { centsToValue, sum, valueToCents } from 'xen-dev-utils'
 import { ceilPow2 } from './utils'
 import { computed, type ComputedRef } from 'vue'
 
+import TIMBRES from '@/timbres.json'
+
+type InharmonicSimpleTimbre = number[]; // Just its partials
+
+type Timbres = {
+  inharmonicSimple: { [key: string]: InharmonicSimpleTimbre }
+  // Allows migration from other data using additional keys and timbre data types
+}
+
+function getInharmonicSimple(id: string): InharmonicSimpleTimbre {
+  const result: any = (TIMBRES as unknown as Timbres).inharmonicSimple[id]
```

Comment:
Just `return (TIMBRES as unknown as Timbres).spectra[id]`. This constant does nothing.